spec/sincerely/generators/aws_ses_webhook_controller_generator_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Sincerely::Generators::AwsSesWebhookControllerGenerator, type: :generator do
+  include Rails::Generators::Testing::Assertions
+
+  let(:name) { 'ses_webhook' }
```

Comment:
Since this is provided by the user, it's up to you to accept my suggestion:
```suggestion
  let(:name) { 'aws_ses_webhook' }
```